.github/workflows/test-all-packages.yml

```diff
@@ -411,15 +506,26 @@ jobs:
           node-version: ${{ steps.vars.outputs.node-version }}
       # END-TEST-BOILERPLATE
       - name: yarn test (SwingSet)
+        if: success() || failure()
         run: cd packages/SwingSet && yarn ${{ steps.vars.outputs.test }} 'test/**/test-[E-Ie-i]*.js' | $TEST_COLLECT
       - name: yarn test (xsnap-lockdown)
+        if: success() || failure()
         run: cd packages/xsnap-lockdown && yarn ${{ steps.vars.outputs.test }} | $TEST_COLLECT
       - name: yarn test (xsnap)
+        if: success() || failure()
         run: cd packages/xsnap && yarn ${{ steps.vars.outputs.test }} | $TEST_COLLECT
       # explicitly test the XS worker, for visibility
       - name: yarn test (SwingSet XS Worker)
-        if: matrix.engine != 'xs'
+        if: success() || failure() && matrix.engine != 'xs'
```

Comment:
maybe `( success() || failure() ) && (matrix.engine != 'xs')` to be clear

Reply:
Absolutely!  This was an incorrect search-and-replace of `always()`.